Signal/src/ViewControllers/CodeVerificationViewController.m

```diff
@@ -260,8 +260,11 @@ - (void)stopActivityIndicator
 - (void)verifyChallengeAction:(nullable id)sender
 {
     [self startActivityIndicator];
+    OWSProdInfo(@"registration_registering_code");
```

Comment:
maybe: `registration_registering_submitted_code`
And separately maybe: `registration_registering_requested_new_code`